scripts/backup

```diff
@@ -19,44 +19,48 @@ ynh_abort_if_errors
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_print_info --message="Loading installation settings..."
 
 app=$YNH_APP_INSTANCE_NAME
 
-final_path=$(ynh_app_setting_get "$app" final_path)
-domain=$(ynh_app_setting_get "$app" domain)
-
-db_name="$app"
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
```

Comment:
Where do you get that? The documentation does not look like this: https://yunohost.org/#/packaging_apps_arguments_management_en. Is the current way deprecated?

Reply:
I follow the example_ynh template...

https://github.com/YunoHost/example_ynh/blob/25251b00d316c41ed1fef6df90312df60615b8da/scripts/backup#L31